MaxMind.Db.Benchmark/project.json

```diff
@@ -0,0 +1,32 @@
+﻿{
+  "version": "2.0.0-*",
+  "description": "Benchmark project to validate .NET reader for the MaxMind DB file format",
+  "packOptions": {
+    "authors": [ "MaxMind, Inc." ],
+    "tags": [ "" ],
+    "projectUrl": "https://github.com/maxmind/MaxMind-DB-Reader-dotnet",
+    "licenseUrl": "https://github.com/maxmind/MaxMind-DB-Reader-dotnet/blob/v2.0.0/LICENSE",
+    "copyright": "Copyright © 2013-2016"
+  },
+
+  "buildOptions": {
+    "emitEntryPoint": true
+  },
+  "dependencies": {
+    "MaxMind.Db": "2.0.0-*"
+  },
+  "frameworks": {
+    "net451": { },
```

Comment:
All three projects are support net45+ (given this chart: https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md#mapping-the-net-platform-standard-to-platforms net45 is compatible with net45x and net46x).